docs/decisions/0005-project-labels.md

```diff
@@ -0,0 +1,154 @@
+# Context and Problem Statement
+
+The goal is to put into place standard labels across all SUSE Rancher projects,
+which allows people from multiple spheres of interest to have a single glance
+into top-level work.
+
+Issues and labels are used for:
+
+- Tracking defects
+- Tracking feature requests
+- Planning releases
+- Managing workflows
+- Reporting stats
+
+The introduction of these changes should have minimum impact on our current
+workflow since we are mainly renaming already existing labels. We use custom
+fields to define [epics][1] & [milestones][2]. We will carry on using them,
+together with the `kind/epic` label, since they are a great way of organizing
+the work.
+
+## Considered Options
+
+Labels can be found here: [https://github.com/aquarist-labs/s3gw/labels][3]
+
+### Issue conventions
+
+All issues should be actionable (within a reasonable timeframe). Usage questions
+and design brainstorming should happen in Github discussions.
```

Comment:
```suggestion
and design brainstorming should happen in GitHub discussions.
```
Please change that in the whole document.

Reply:
Updated